src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncTests.cs

```diff
@@ -3400,27 +3400,44 @@ class Program
             CompileAndVerify(comp.WithOptions(TestOptions.ReleaseExe));
         }
 
+        /// <summary>
+        /// await should be disallowed in top-level
+        /// script statements since the script main
+        /// method should complete synchronously.
+        /// </summary>
```

Comment:
Logged #4495 to change the Script entry point to block until the async initializer completes.